### ISSUE TITLE
RTL is not supported

### ISSUE THREAD

**Issue by hpardess** [sanjayV/ng-image-slider#6]
<img width="551" alt="Screen Shot 1398-02-09 at 1 41 37 AM" src="https://user-images.githubusercontent.com/6196491/56870138-fb0d6400-6a1f-11e9-9467-e429aedc64e5.png">

**Comment by sanjayV** [comment]
Hi @hpardess 
can you describe more what exact issue are you facing?

**Comment by sanjayV** [comment]
Hi @hpardess 
It's not possible for me to fix issue without looking your code, so it would good if you provide full code snippet where you using RTL.

**Comment by SamihIbrahim** [comment]
I think I can elaborate a bit more since I am having the same issue. When the image slide is used in an RTL based language (Arabic in my case), the next and prev arrow would completely overlap one another making the one at the bottom in accessible. I am not doing any customization to the lib code. Sample as below:
`<div class="tab-content text-center">
        <div class="tab-pane active" *ngIf="imageObject" name="images" role="tabpanel">
          <ng-image-slider [images]="imageObject" [imagePopup]="true" [animationSpeed]="0.2"></ng-image-slider>
        </div>
      </div>`

**Comment by sanjayV** [comment]
Hi @SamihIbrahim 
Thanks for elaborate issue.
I will try to add this new feature for you guys in next version.

**Comment by SamihIbrahim** [comment]
Thanks @sanjayV. As a workaround, I set the dir attribute of the div just outer of the ng-image-slider to rtl. This fixes the issue and restores the slide to a navigable state. Again thanks for the quick responses and excellent slider.

**Comment by sanjayV** [comment]
@hpardess @samihibrahim 
I have released new new version (1.3.1). Added new **@input: direction** option  for set text direction
example:
```
<ng-image-slider [images]="imageObject" direction="rtl" #nav></ng-image-slider>
```

for more info please visit [link](https://www.npmjs.com/package/ng-image-slider#api-reference-optional-)

**Comment by sanjayV** [comment]
@hpardess 
I am closing this issue. I have fixed this issue and not getting any response from your side.
